docs/source/development/debugging.md

```diff
@@ -1,83 +1,9 @@
 # Debugging
 
-## Introduction
+> [!NOTE]
```

Comment:
You can use the standard admonition for this. Take a look at the tutorial for an example of how to use it if you're not sure.